include/core/jbackend.h

```diff
@@ -126,6 +126,13 @@ struct JBackend
 	JBackendType type;
 	JBackendComponent component;
 
+	/**
+	 * delete everything in the backend
+	 * \param[in] which namespace to delete
+	 * \param[out] how many "things" are deleted within the specified namespace
```

Comment:
There are no in and out parameters in the actual function below.

Reply:
gpointer is an input parameter containing a batch structure, Error is a out parameter, where the error message-string may contain the number of elements which are deleted